### PR TITLE
completed delete function and added button. Ready for code review.

### PR DESCRIPTION
**Pull request by cwoodwarduk**
Ready for code review

### CODE REVIEW

File: public/js/eventsPage.js [hunk 1]
```diff
@@ -97,13 +97,44 @@ function displayEventsHandler(eventsAndHiringPartners) {
...
+    let hpDeleteForms = document.querySelectorAll(`.hiring-partner input[data-event='${event.id}'`)
+    console.log(hpDeleteForms)
+    hpDeleteForms.forEach(function(hpDelete){
+        hpDelete.addEventListener('click', function(e){
+            e.preventDefault()
+            let data = {
+                event_id: e.target.dataset.event,
+                hp_id: e.target.dataset.hp
+            }
+            console.log(data)
```
Comment (sudokufan) [inline public/js/eventsPage.js]: Presumably don't need this console log?

File: public/js/eventsPage.js [hunk 2]
```diff
@@ -97,13 +97,44 @@ function displayEventsHandler(eventsAndHiringPartners) {
     }
 };
 
-async function displayEvents(events, hiringPartners) {
+async function displayEvents(events, hiringPartners){
```
Comment (sudokufan) [inline public/js/eventsPage.js]: needs doc block

File: public/js/eventsPage.js [hunk 3]
```diff
@@ -97,13 +97,44 @@ function displayEventsHandler(eventsAndHiringPartners) {
...
-        .then(async (event) => {
-            await displayHiringPartnersAttending(event)
+        await eventGenerator(event, hiringPartners).then(event => {
+            displayHiringPartnersAttending(event)
+        })
+        return event
+    })
+}
+
+async function addEventListenersToHpDelete(event){
```
Comment (sudokufan) [inline public/js/eventsPage.js]: doc blocks

File: public/js/eventsPage.js [hunk 4]
```diff
@@ -97,13 +97,44 @@ function displayEventsHandler(eventsAndHiringPartners) {
...
+    let hpDeleteForms = document.querySelectorAll(`.hiring-partner input[data-event='${event.id}'`)
```
Comment (sudokufan) [inline public/js/eventsPage.js]: why the square bracket [ ? is that a typo?

Comment (cwoodwarduk) [reply public/js/eventsPage.js]: Done

File: public/js/eventsPage.js [hunk 5]
```diff
@@ -99,11 +98,42 @@ function displayEventsHandler(eventsAndHiringPartners) {
...
+    console.log(hpDeleteForms)
```
Comment (sudokufan) [inline public/js/eventsPage.js]: console log

File: public/js/eventsPage.js (hunk 1 above)
Comment (cwoodwarduk) [reply public/js/eventsPage.js]: Done.

**Pull request merged**